e2e-tests/monitoring-2-0/compare/statefulset_monitoring-pxc-oc.yml

```diff
@@ -170,6 +170,17 @@ spec:
         - mountPath: /etc/mysql/vault-keyring-secret
           name: vault-keyring-secret
       dnsPolicy: ClusterFirst
```

Comment:
I think entrypoint.sh also needed for this file

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/408/commits/84533f38db5e68f75064af81973a65cea1cde512